package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@learnersguild/game-cli",
-  "version": "0.5.0",
+  "version": "0.7.0",
```

Comment:
Yeah ... there were multiple branches going in parallel and I was trying to avoid collisions. Kinda silly -- would have likely been caught in a merge. :-/